molecule/default/tests/test_reboot_devices.py

```diff
@@ -0,0 +1,58 @@
+import os
+import re
+import pytest
+import testinfra.utils.ansible_runner
+
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('dashboard_hosts')
+
+pre_cmd = "sudo bash -c \"source /root/openrc; "
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_nova_compute(host):
+    cmd = pre_cmd + "ansible -m shell nova_compute -a \'reboot\' \""
+    assert host.run_expect([0], cmd)
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_log_hosts(host):
+    cmd = pre_cmd + "ansible -m shell log_hots -a \'reboot\' \""
+    assert host.run_expect([0], cmd)
+    verify_the_device_is_backup(host)
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_infra03(host):
+    cmd = pre_cmd + "ansible -m shell <DEV#>-infra03 -a \'reboot\' \""
```

Comment:
What is `<DEV#>`?

Reply:
I am not sure exactly how the partition looks like in MNAIO, could be '/dev', but I am not sure so just put that '<DEV#> in and will replace it when we have a better look at the MNAIO env.